### PR TITLE
integrate

### PR DESCRIPTION
**Pull request by senghuot**

### CODE REVIEW

File: Atomizer/src/main/java/senghout/github/com/atomizer/AtomizerController.java [hunk 1]
```diff
@@ -13,23 +13,27 @@
...
-        url.setTinyUrl(shortURL + Math.random());
-        repo.save(url);
-        return shortURL;
+    public String addUrl(@RequestBody TinyUrl tinyUrl) {
+        tinyUrl.tinyUrl = AtomizerUtils.encodeNumber(Math.abs(tinyUrl.fullUrl.hashCode()));
+        repo.save(tinyUrl);
+
+        System.out.println("-----------------------------");
+        System.out.println(tinyUrl);
+        System.out.println("-----------------------------");
```
Comment (chang47) [inline Atomizer/src/main/java/senghout/github/com/atomizer/AtomizerController.java]: Remove println statements

File: Atomizer/src/main/java/senghout/github/com/atomizer/AtomizerUtils.java [hunk 2]
```diff
@@ -6,8 +6,10 @@ public static String encodeNumber(int number) {
         int size = letters.length();
         int count = 6;
         char[] shortURL = new char[7];
+        System.out.println(number);
```
Comment (chang47) [inline Atomizer/src/main/java/senghout/github/com/atomizer/AtomizerUtils.java]: Here too

File: Atomizer/src/main/java/senghout/github/com/atomizer/model/TinyUrl.java [hunk 3]
```diff
@@ -0,0 +1,19 @@
+package senghout.github.com.atomizer.model;
+
+public class TinyUrl {
+
+    public String tinyUrl;
```
Comment (chang47) [inline Atomizer/src/main/java/senghout/github/com/atomizer/model/TinyUrl.java:9]: Don't we need the @Id annotation?

**Review by chang47 — CHANGES_REQUESTED** [review]
Added some nit comments.

Can we also have a better title, description, and attach the appropriate issue to this this PR?

File: Atomizer/src/main/java/senghout/github/com/atomizer/AtomizerUtils.java (hunk 2 above)
Comment (senghuot) [reply Atomizer/src/main/java/senghout/github/com/atomizer/AtomizerUtils.java]: removed

**Review by chang47 — APPROVED** [review]
Looks good!

**Pull request merged**